11/29/2022 - 5:30 (UTC -7)

Starting install

OS: FreeBSD 13.1-RELEASE

PKGs to install (as tested):
-py39-dj-database-url-1.0.0
-py39-django41-4.1.2
-py39-sentry-sdk-1.5.12
-py39-pydantic-1.10.2
-postgresql14-server-14.5
-postgresql14-client-14.5
-"allow.sysvipc=1" in jail properties to run Postgresql
-py39-pip-22.2.2
-pip install pydantic[email]
-pip install django-htmx
-pip install pyld
-libxslt (PKG)
-pip install psycopg2

***Additional Notes***
all pip commands were run as a "regular" user in the wheel group. Pip was installed
via PKG as well as django.

***Code Added***
settings.py - line 94:
added host (jail) IP for PGHOST
settings.py - line 100:
"allow_reuse=True"


All the above modules (pip) and packages (pkg) installed as expected. However,
when starting the application with the following command:
"python3.9 manage.py runserver <ip>:8080"
produces the following error:
  File "/usr/home/<user>/activities/models/fan_out.py", line 23
    match (fan_out.type, fan_out.identity.local):
                                                 ^
SyntaxError: invalid syntax

